app/controllers/group_settings_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, CVP Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class GroupSettingsController < ModalCrudController
```

Comment:
Du überschreibst hier viel dry_crud (authorisierung, find, assign). In solchen fällen fände ich es besser, wenn wir direkt von ApplicationController erben würden

Reply:
das stimmt!
es hat mir aber sehr geholfen Dry Crud besser zu verstehen :wink: